### ISSUE TITLE
Computer object does not get properly created

### ISSUE THREAD

**Issue by milanche1311** [sqlcollaborative/PSDatabaseClone#44]
**Before submitting the bug**
- [x] Ensure you are able to reproduce it on the latest released version (_we release often_)
- [x] Verified this bug is not already reported in an issue
- [x] Verified errors are not related to permissions
- [x] Can reproduce in a clean PowerShell session (_clean = `powershell -NoProfile`_)

**Describe the bug**
In New-PSDCClone, the $computer object does not get correct value for computer name and create clone vhd fails

**To Reproduce**

There is no property named "Name" for the server object:

```powershell
<#
 # Setup the computer object
  $computer = [PsfComputer]$server.Name
#>
```

**Expected behavior**

Changing the code like this:

  $computer = [PsfComputer]$server.ComputerName

gives the proper value

**Screenshots**

As it can be seen, the $server object has the property named "ComputerName" and not "Name". 

![image](https://user-images.githubusercontent.com/42850146/52450623-2462d600-2b3c-11e9-9c23-9290bb47d0e5.png)

**Environmental data**

 - PowerShell: 
![image](https://user-images.githubusercontent.com/42850146/52450693-71df4300-2b3c-11e9-9c6e-e153c25c6f5e.png)

**Comment by ferdiozturk** [comment]
and even if you take .ComputerName, then $computer.IsLocalhost doesn't exist

**Comment by SQLcyclopedia** [comment]
I'm having the same issue as New-PSDCClone (dev branch) doesn't handle named instances and Connect-DbaInstance function therefore fails.

Have created a fix branch but unable to publish/make pull request.

**Comment by sanderstad** [comment]
What's the problem to create the PR?

**Comment by ferdiozturk** [comment]
I added .ComputerName + IsLocalhost to my fork repo, if anyone wants to use it. Works for me

**Comment by sanderstad** [comment]
If you can create a PR from your fork that would be great

**Comment by SQLcyclopedia** [comment]
I’m not sure this is possible. Are you able to branch the dev from sqlcollab account, commit your changes there and submit pull request? (perhaps it’s just me that can’t!)

**Comment by sanderstad** [comment]
I should have mentioned that, that message was meant for @ferdiozturk

**Comment by ferdiozturk** [comment]
@sanderstad sorry! Sure, I will do that some time next week then. I want to test one or two things more. Thanks

**Comment by SQLcyclopedia** [comment]
Remove-PSDCClone also needs some work to be able to handle named instances locally and remotely. Have you considered this?

**Comment by bertvanlandeghem** [comment]
@ferdiozturk Any progress on the PR? I'd be happy to fork and PR if you don't have the time.

BTW, the 'IsLocalHost' part appears to work with the current release, no need to add it, IMHO.
`
PS C:\> $server = Connect-DbaInstance -SqlInstance MYLAPTOP\BVLTEST1 -SqlCredential $credential
PS C:\> $computer = [PsfComputer]$Server.ComputerName
PS C:\> $computer

ComputerName IsLocalhost    Type InputObject
------------ -----------    ---- -----------
MYLAPTOP           True Default MYLAPTOP
`

**Comment by sanderstad** [comment]
@bertvanlandeghem  If you could take a look. I've been so terribly busy lately that I have not had the time.

**Comment by bertvanlandeghem** [comment]
Fixed for New-PSDCClone in #46 , but as @SQLcyclopedia mentioned:
> Remove-PSDCClone also needs some work to be able to handle named instances locally and remotely. 
> Have you considered this?
I'll have a look at that next week.

**Comment by sanderstad** [comment]
I have made several changes and pushed to the development branch. 
I have not been able to create a well enough test environment to get the same kind of errors.
If you could test it from the development branch and tell me if you still have the problem, that would be great.
